msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -782,7 +782,7 @@ Enables your app in non-standard channels. If your app supports a team scope and
 > [!NOTE]
 >
 > * If your app supports a team scope, it functions in the standard channels regardless of the values that are defined in this property.
-> * Your app can account for the unique properties of each of the channel types to function properly. To enable your tab for private and shared channels, see [retrieve context in private channels](~/tabs/how-to/access-teams-context.md#retrieve-context-in-private-channels) and [retrieve context in shared channels](~/tabs/how-to/access-teams-context.md#retrieve-context-in-microsoft-teams-connect-shared-channels).
+> * Your app can account for the unique properties of each of the channel types to function properly. To enable your tab for private and shared channels, see [retrieve context in private channels](~/tabs/how-to/access-teams-context.md#retrieve-context-in-private-channels) and [get context in shared channels](../../tabs/how-to/access-teams-context.md#get-context-in-shared-channels)
```

Comment:
Shall we mention min manifest version: 1.13

Reply:
Min manifest version 1.13 is covered [here](https://review.docs.microsoft.com/en-us/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?branch=pr-en-us-6846&tabs=manifest-teams-toolkit#update-the-app-manifest) and [here](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?branch=pr-en-us-6846&tabs=javascript%2Cmanifest-teams-toolkit)